README.md

```diff
@@ -2,3 +2,13 @@
 
 This is the repository that we maintain the default DAX schema files for
 [PyDAX](https://github.com/codait/pydax).
+
+To run tests locally, run:
+
+    $ pip install -U -rrequirements/tox.txt  # If you are inside a virtual environment, conda environment
```

Comment:
Extra r in requirements

Reply:
There's a missing space between `-r` and `requirements`. Added